tests/data/ApPipe.yaml

```diff
@@ -1,12 +1,11 @@
 description: End to end Alert Production pipeline specialized for HiTS-2015
 #
-# This file was copied from ap_verify_ci_hits2015 branch DM-34237 and modified
+# This file was copied from ap_verify_ci_hits2015/pipelines/ApPipe.yaml
+# with doSolarSystemAssociation disabled and modified
```

Comment:
There's a bit of ambiguous parsing here. Perhaps "and modified, with solar system association disabled"?

Out of curiosity, what other modifications were made?

Reply:
Bad english. This file is the same as `ap_verify_ci_hits2015/pipelines/ApPipe.yaml` except 
1) `doSolarSystemAssociation` is `false` instead of `true` 
2)  `$PROMPT_PROTOTYPE_DIR/pipelines/calibrate.py` instead of `$AP_VERIFY_CI_HITS2015_DIR/config/calibrate.py` . 
I'll rephrase.